src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -42,19 +42,37 @@ public CommandResult execute() {
 
     /**
      * Retrieves all persons in the address book whose names contain some of the specified keywords.
+     * Keywords need not be case sensitive.
      *
      * @param keywords for searching
      * @return list of persons found
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
+        final Set<String> lowerCaseKeywords = lowerCaseAllWords(keywords);
```

Comment:
You might want to look into whether[ Java String APIs](https://docs.oracle.com/javase/9/docs/api/java/lang/String.html#equalsIgnoreCase-java.lang.String-) can help you to do this

Reply:
Thanks for the suggestion. I had totally forgotten that I can use equalsIgnoreCase.